file2.js

```diff
@@ -0,0 +1,558 @@
+        $teamBox.prop("disabled", true).selectpicker("refresh");
+        utils.request(teamsUrl, successFallback, failureFallback, function() {
+          $teamBox.selectpicker("refresh");
+          utils.selectpickerAddIdtoList($teamBox, "qa-teams");
+        });
+        utils.selectpickerAddIdtoList($orgBox, "qa-organization");
+      });
+
+      $("#transferBtn").click(function() {
+        //Update transfer project confirmation modal text
+        var confirmationBox = $(".modal-transfer-project .modal-body");
+        var projectName = confirmationBox.data("projectname");
+
+        var selectedTeams = [];
+        $teamBox.find("option:selected").each(function(i, e) {
+          selectedTeams.push($(e).text());
+        });
+        if (selectedTeams.length > 0) {
+          $(".btnTransferProject").show();
+          var baseMsg = "Are you sure that you want to transfer <strong>" + projectName + "</strong>";
+          var orgName = "<strong>" + $orgBox.find("option:selected").text() + "</strong>";
+          var teamNames = "";
+          for (var i = 0; i < selectedTeams.length; i++) {
+            teamNames += " <strong>" + selectedTeams[i] + "</strong>";
+          }
+
+          if (teamNames !== "") {
+            teamNames = " - " + teamNames
+          }
+
+          confirmationBox.find("p").html(baseMsg + " to " + orgName + teamNames + "?");
+        } else {
+          $(".btnTransferProject").hide();
+          confirmationBox.find("p").html("You need to choose at least one team.");
+        }
+
+
+      });
+
+      var successFallback = function(jsResponse) {
+        teamBoxData = [];
+        jsResponse.data.forEach(function(element) {
+          var option = "<option value='"+ element.id +"'>" + element.text + "</option>";
+          teamBoxData.push(option);
+        });
+        //teams is a array of {id, text} with the id and name of the team
+        $teamBox.html(teamBoxData);
+        if(jsResponse.data.length > 1) {
+          $teamBox.prop("multiple", true);
+          $teamBox.selectpicker("destroy");
+          $teamBox.selectpicker({
+            iconBase: 'fa',
+            tickIcon: 'fa-check'
+          });
+        } else {
+          $teamBox.prop("multiple", false);
+          $teamBox.selectpicker("destroy");
+          $teamBox.selectpicker();
+        }
+
+        $teamBox.prop("disabled", false);
+      };
+
+      var failureFallback = function() {
+        teamBoxData = [];
+      };
+
+      //Gets the teams for the selected organization
+      $orgBox.selectpicker("refresh");
+    };
+
+    ProjectSettings.prototype.bindUpdateProjectName = function(projectId) {
+      $(".BtnUpdateProjectName").click(function() {
+        var url = $(this).data("url");
+        var postVariable = {};
+        postVariable.projectName = $(".TxtUpdateProjectName").val();
+        postVariable.projectId = projectId.toString();
+
+        utils.JSend(url, postVariable, function(data) {
```

Comment:
![Codacy](https://app.dev.codacy.org/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor. (new-cap)](https://app.dev.codacy.org/app/codacytestuser/CheckPrCommentsPublicProject/pullRequest?prid=7992)